### PR TITLE
IA1-4618_hypothesis-styles-in-SEB

### PR DESCRIPTION
**Pull request by andreyshepilov**

### CODE REVIEW

**Comment by sergeyvdovareize** [comment]
Let's clean comments before merging

**Comment by andreyshepilov** [comment]
> Let's clean comments before merging

Done

**Review by sergeyvdovareize — APPROVED** [review]
just removed unused css files

File: src/sidebar/components/test/thread-list-test.js [hunk 1]
```diff
@@ -235,15 +235,6 @@ describe('ThreadList', () => {
...
 
-    it('only renders visible threads', () => {
-      const wrapper = createComponent();
-      const renderedThreads = wrapper.find('ThreadCard');
-
-      // "5" is the current expected value given the thread heights, scroll
-      // container size and constants in `../util/visible-threads`.
-      assert.equal(renderedThreads.length, 5);
-    });
-
```
Comment (sergeyvdovareize) [inline src/sidebar/components/test/thread-list-test.js:246]: this rest is not relevant anymore because we don't apply css on the hypothesis side

**Review by sergeyvdovareize — APPROVED** [review]

**Pull request merged**